tests/vmi_lifecycle_test.go

```diff
@@ -996,6 +1005,43 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 					fmt.Sprintf("Expected CPU model to equal to the default (%v)", v1.DefaultCPUModel),
 				)
 			})
+			It("[sig-compute]should add node selector to virt-launcher when setting default cpuModel in kubevirtCR", func() {
+				if len(supportedCpuModels) < 1 {
+					Skip("Must have at least one supported cpuModel for this test")
+				}
+				defaultCPUModel := supportedCpuModels[0]
+				config := originalConfig.DeepCopy()
+				config.CPUModel = defaultCPUModel
+				tests.UpdateKubeVirtConfigValueAndWait(*config)
+
+				newVMI := newCirrosVMI()
+				newVMI = tests.RunVMIAndExpectLaunch(newVMI, 90)
+				By("Fetching virt-launcher pod")
+				virtLauncherPod := tests.GetRunningPodByVirtualMachineInstance(newVMI, newVMI.Namespace)
+				Expect(virtLauncherPod.Spec.NodeSelector).To(HaveKey(ContainSubstring(defaultCPUModel)), "Node selector for the cpuModel in vmi spec should appear in virt-launcher pod")
+
+			})
+
+			It("[sig-compute]should prefer node selector of the vmi if cpuModel field is set in kubevirtCR and in the vmi ", func() {
```

Comment:
here as well + you have a redundant space at the end

Reply:
done